### PR TITLE
Enable creation of Imovo digital voucher

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This responds to a call to create a digital voucher by calling out to the Imovo API for a card code and a letter code, and responding with the codes in a `Voucher` instance.

### CODE REVIEW

File: handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala [hunk 1]
```diff
@@ -7,40 +7,92 @@ import cats.implicits._
...
+    ): EitherT[F, DigitalVoucherApiException, Voucher] = {
+
+      def requestVoucher(code: CampaignCodeSet) = {
+        val tomorrow = LocalDate.now.plusDays(1)
+        (
+          imovoClient.createVoucher(subscriptionId, code.card, tomorrow).leftMap(List(_)),
+          imovoClient.createVoucher(subscriptionId, code.letter, tomorrow).leftMap(List(_))
+          ).parMapN { (cardResponse, letterResponse) =>
+          Voucher(cardResponse.voucherCode, letterResponse.voucherCode)
+        }.leftMap(es => DigitalVoucherApiException(es.head))
```
Comment (arausuy) [inline handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala]: Does this return both potential errors or just the first one?

File: handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala [hunk 2]
```diff
@@ -7,40 +7,92 @@ import cats.implicits._
...
+    )
+    val sixDayCampaignCodes = CampaignCodeSet(
+      CampaignCode("GMGSub6DayCard"),
+      CampaignCode("GMGSub6DayHNDSS")
+    )
+
+    Map(
+      RatePlanName("Everyday") -> everydayCampaignCodes,
+      RatePlanName("Everyday+") -> everydayCampaignCodes,
+      RatePlanName("Saturday") ->
```
Comment (arausuy) [inline handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala]: afaik we're not offering Saturday & Saturday+ anymore - I'll follow this up

File: handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala (hunk 1 above)
Comment (kelvin-chappell) [reply handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala]: Now it returns all errors.

File: handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala (hunk 2 above)
Comment (kelvin-chappell) [reply handlers/digital-voucher-api/src/main/scala/com/gu/digital_voucher_api/DigitalVoucherService.scala]: I've removed the Saturday rate plans for now.  But they can easily be reinstated later if necessary.

**Pull request merged**